src/main/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTiming.java

```diff
@@ -169,7 +169,8 @@ public static GenericStatus computeChunkStatus(@Nonnull WorkflowRun run,
                         return (lastNode.getError() == null) ? GenericStatus.SUCCESS : GenericStatus.FAILURE;
                     }
                 }
-                return (isPendingInput(run)) ? GenericStatus.PAUSED_PENDING_INPUT : GenericStatus.IN_PROGRESS;
+                return (isPendingInput(run) && lastNode.getAction(PauseAction.class) != null)
```

Comment:
I am honestly not sure what I was thinking when I wrote this code, but the fix looks right to me.